dotnet-desktop-guide/net/winforms/toc.yml

```
@@ -103,5 +103,9 @@ items:
   items:
   - name: Overview
     href: printing/overview.md
-  - name: Print a multi-page text file
-    href: printing/how-to-print-text-document.md
+  - name: Common tasks
+    items:
+    - name: Print a text document
+      href: printing/how-to-print-a-text-document.md
```

Whoops, I made a mistake.
```suggestion
      href: printing/how-to-print-text-document.md
```